great_expectations/data_context/store/gx_cloud_store_backend.py

```diff
@@ -664,3 +624,107 @@ def _validate_key(self, key) -> None:
             raise TypeError(
                 f"The provided resource_type {resource_type} is not a valid GXCloudRESTResource"
             )
+
+    @classmethod
+    def construct_versioned_url(
+        cls,
+        base_url: str,
+        organization_id: str,
+        resource_name: str,
+        id: Optional[str] = None,
+    ) -> str:
+        """Construct the correct url for a given resource."""
+        version = cls._ENDPOINT_VERSION_LOOKUP.get(resource_name, EndpointVersion.V0)
+
+        if version == EndpointVersion.V1:
+            url = urljoin(
+                base_url,
+                f"api/v1/organizations/{organization_id}/{hyphen(resource_name)}",
+            )
+            if id:
+                url = f"{url}/{id}"
+        else:  # default to EndpointVersion.V0
+            url = urljoin(
+                base_url,
+                f"organizations/{organization_id}/{hyphen(resource_name)}",
+            )
+            if id:
+                url = f"{url}/{id}"
```

Comment:
```suggestion
        if version == EndpointVersion.V1:
            url = urljoin(
                base_url,
                f"api/v1/organizations/{organization_id}/{hyphen(resource_name)}",
            )
        else:  # default to EndpointVersion.V0
            url = urljoin(
                base_url,
                f"organizations/{organization_id}/{hyphen(resource_name)}",
            )
        if id:
            url = f"{url}/{id}"
```

Reply:
definitely clearer, thanks! updated in [15e4b04](https://github.com/great-expectations/great_expectations/pull/9651/commits/15e4b04fc2dbfdc0cc6cf271e171261abb8d9cf4)